src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -362,7 +362,7 @@ def test_vm_capture_zone_resilient_image(self, resource_group):
             'vm': 'vm-generalize'
         })
 
-        self.cmd('vm create -g {rg} --location {loc} -n {vm} --admin-username ubuntu --image centos --admin-password testPassword0 --authentication-type password --nsg-rule NONE')
+        self.cmd('vm create -g {rg} --location {loc} -n {vm} --admin-username ubuntu --image CentOS85Gen2 --admin-password testPassword0 --authentication-type password --nsg-rule NONE')
```

Comment:
Because we need to modify a large number of test cases, it may block PR merge in this sprint, and PS team had tested these new aliases are valid, we can consider separately submitting another PR to modify these tests to unblock this PR merge~

Reply:
Received, already modified in https://github.com/Azure/azure-cli/pull/25930/commits/aa5faa0f9633d2e9989acd67913f7f4951921e80 and PR test splited in https://github.com/Azure/azure-cli/pull/25956